rosidl_typesupport_connext_cpp/cmake/rosidl_typesupport_connext_cpp_generate_interfaces.cmake

```diff
@@ -16,7 +16,7 @@ set(_ros_idl_files "")
 foreach(_idl_file ${rosidl_generate_interfaces_IDL_FILES})
   get_filename_component(_extension "${_idl_file}" EXT)
   # Skip .srv files
-  if("${_extension}" STREQUAL ".msg")
+  if("${_extension} " STREQUAL ".msg ")
```

Comment:
Any condition in CMake is subject to potential double evaluation. The white space guaranteed that that does not happen since a whitespace can never be part of a variable name.